src/OSPSuite.Infrastructure.Import/Core/MissingColumnException.cs

```diff
@@ -8,5 +8,9 @@ public class MissingColumnException : AbstractImporterException
       public MissingColumnException(IReadOnlyList<string> missingColumns) : base(Error.MissingColumnException(missingColumns))
       {
       }
+
+      public MissingColumnException(string missingColumn) : base(Error.MissingColumnException(new List<string>() { missingColumn}))
```

Comment:
Ok
Better. Call this(new list...) That way you do not have to duplicate the logic.
Merging.